configure.ac

```diff
@@ -179,29 +179,6 @@ AS_CASE(["x$srcdir"],
 ])
 AC_MSG_NOTICE(AS_HELP_STRING([Value of _srcdir_abs:], ['$_srcdir_abs']))
 
-# This script is part of the sources, so we expect it to be available
```

Comment:
Is there a particular reason to remove this bit (git_details script from configure)? Was it moved elsewhere, or just dropped?